### DISCUSSION TITLE
Sending emails for password recovery

### DISCUSSION THREAD

**Discussion by alb3rtod** [chartbrew/chartbrew#249] [Q&A] [answered]
Is it necessary to also set up an SMTP server container for user password recovery emails to be sent successfully? If not, what can I do to ensure that the emails are sent successfully? I tried sending a password recovery email, but it never arrived. Thank you.

**Comment by razvanilin** [comment, marked as answer]
Hey @alb3rtod you will need to set up the variables below to be able to send. You will have to use an existing email address with an existing smtp server to make it work:

![CleanShot 2024-10-23 at 11 29 54@2x](https://github.com/user-attachments/assets/359dcd36-2108-4960-8152-b0a9d3253a6f)

**Reply by alb3rtod** [reply]
Hi @razvanilin 

I set the variables as follows:

![{E6866C6A-8656-4C5D-A53A-5A432AF5E993}](https://github.com/user-attachments/assets/579ae7f8-c70e-4835-a785-2695d7ed5149)

But I'm facing the following error:

![{0D5BAA5C-B432-4135-869B-97D2065AEF25}](https://github.com/user-attachments/assets/d4aabc8d-458b-4392-8abb-309acde8b155)

What could I probably be doing wrong? The admin email exists, and I am able to log into the email account.

**Reply by razvanilin** [reply]
I just had a look at Yahoo authentication and it looks like you have to create a specific app password to authenticate. Have you done that?

https://help.yahoo.com/kb/SLN15241.html

**Reply by alb3rtod** [reply]
Hi @razvanilin 

It seems like Yahoo is having some issues with third-party apps. I connected to another public SMTP server and was able to send the emails successfully.

Thank you so much.
